docs/contributorfile.rst

```diff
@@ -185,13 +209,16 @@ To be merged, PRs have to:
 
 As we’re trying to maintain at least a 90% code coverance, you’re strongly encouraged to write all the necessary tests not to drop below the threshold. If our coverance becomes too low, you might be asked to add more tests and/or your PR might be rejected.
 
```

Comment:
I think we should always ask for a test in cas there is a new feature

Reply:
or function

Reply:
Maybe I need to rephrase it, but isn't it what we're stating the line before this?

Reply:
true